src/lib/Scenes/Map/GlobalMap.tsx

```diff
@@ -420,12 +420,5 @@ export const GlobalMapContainer = createRefetchContainer(
         }
       }
     }
-  `,
-  graphql`
-    query GlobalMapRefetchQuery($near: Near!, $maxInt: Int!) {
-      viewer {
-        ...GlobalMap_viewer @arguments(near: $near, maxInt: $maxInt)
-      }
-    }
```

Comment:
@l2succes I also went ahead and removed this.

https://artsy.slack.com/archives/CFNFWD5F0/p1551809664119900